lib/net/ldap.rb

```diff
@@ -325,7 +325,7 @@ class Net::LDAP
 
   universal = {
     constructed: {
-      107 => :array, #ExtendedResponse (PasswdModifyResponseValue)
+      107 => :string, # ExtendedResponse
```

Comment:
Why change this?

Reply:
Without this change in place, the reply would cause an exception to be raised when it was being parsed.

Reply:
I guess this is fine if it doesn't break other queries